openapi.yaml

```diff
@@ -0,0 +1,675 @@
+# This is the Open API interface for Open Service Broker API. Every attempt will
+# be made to make the Open API version of OSB API accurately represent the
+# written specification. If the spec and this document conflict, the spec is
+# the authroity.
+
+openapi: "3.0.0"
+
+info:
+  title: Open Service Broker API
+  description: The Service Broker API defines an HTTP(S) interface between Platforms and Service Brokers.
+  license: 
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  contact: 
+    name: Open Service Broker API
+    url: https://www.openservicebrokerapi.org/
+    email: open-service-broker-api@googlegroups.com
+  version: v2.13
+
+servers:
+  - url: http://example.com
+    description: Broker Endpoint
+  - url: http://localhost:3000
+    description: OSB Checker - MockOSB
+
+paths:
+  /v2/catalog:
+    get:
+      summary: get the service catalog
+      tags:
+        - Catalog
+      operationId: catalog.get
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+      responses:
+        '200':
+          description: catalog response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Catalog'
+
+  /v2/service_instances/{instance_id}:
+    put:
+      summary: provision a service instance
+      tags:
+        - ServiceInstances
+      operationId: serviceInstance.provision
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+        - $ref: '#/components/parameters/OriginatingIdentity'
+        - name: instance_id
+          in: path
+          description: instance id of instance to provision
+          required: true
+          schema:
+            type: string
+        - name: accepts_incomplete
+          in: query
+          description: asynchronous operations supported
+          schema:
+            type: boolean
+      requestBody:
+        description: parameters for the requested service instance provision
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ServiceInstanceProvisionRequest'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceInstanceProvision'
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceInstanceProvision'
+        '202':
+          description: Accepted
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AsyncOperation'
```

Comment:
HTTP 202 responses should include the `dashboard_url` as this is the only time it can be returned in the current spec

Reply:
This was a mistake, AsyncOperation was suppose to be used for only the update, not the create.

Reply:
👍